transcom/mymove
# INT B-17960 UI fixes

## [I-12660](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A925256)
## [I-12655](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A924620)

## Summary

This PR fixes the above two issues by fixing typos, adding an endpoint in the backend to get only the PPM actual weight, and adding code to the frontend to ensure we are only getting the current PPM's net weight when on the "Review Documents" screen.

### How to test

Follow the steps in #12072, but instead make sure to make a move with at least 1 HHG shipment and 1 PPM shipment. When on the "Review Documents" screen as an SC, verify that the "Actual Net Weight" field shows only the net weight for the current PPM shipment, and does not include the HHG shipment in its calculation.
